enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/validator/JWTValidator.java

```diff
@@ -70,7 +70,7 @@ public JWTValidationInfo validateJWTToken(SignedJWTInfo signedJWTInfo) throws En
         }
         jwtValidationInfo.setValid(false);
         jwtValidationInfo.setValidationCode(APIConstants.KeyValidationStatus.API_AUTH_INVALID_CREDENTIALS);
-        logger.error("No matching issuer found for the token with issuer : " + issuer);
+        logger.debug("No matching issuer found for the token with issuer : " + issuer);
```

Comment:
As for this, since it is about the issuer, shall we keep it? @pubudu538 @VirajSalaka @AmaliMatharaarachchi

Reply:
+1. We have no other way to track this down I guess. And this seems to be less likely to occur. Let's make it an info log anyway.